README.md

```diff
@@ -0,0 +1,3 @@
+# Memory-Game
+
+This is a Memory-Game created using HTML, CSS and JavaScript. The objective of the game is to check whether the cards the user selects matches or not. If 2 cards match, then the card are removed from the set. If they don't match, then they are once again return to the same position as before
```

Comment:
It isn't necessarily 2 cards always. We can match any number of cards based on the difficulty level